.travis.yml

```diff
@@ -3,8 +3,9 @@ python:
   - "3.6"
   - "3.7"
   - "3.8"
+  - "3.9"
 install:
-  - pip install codecov tox-travis
+  - pip install --upgrade setuptools importlib-metadata codecov tox-travis
```

Comment:
Without this I wan't able to pass the Travis build on 3.7 due to some `setuptools` vs `importlib-metadata` incompatibility, `tox` wasn't able to import `setuptools`.